deploy/platform/kubernetes/feature-rover/profiling.yaml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ebpf-profiling-trgigger
+spec:
+  schedule: "5 * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          containers:
+            - name: ebpf-profiling-trigger
+              image: ${SW_CLI_IMAGE}
+              volumeMounts:
+              command:
+                - /bin/sh
+                - -c
+                - >
+                  /swctl --base-url=http://oap:12800/graphql profiling ebpf create fixed \
+                    --service-name agentless::recommendation --labels mesh-envoy --duration=9m;
+                  /swctl --base-url=http://oap:12800/graphql profiling ebpf create fixed \
+                    --service-name agentless::app --labels mesh-envoy --duration=9m;
```

Comment:
Did you verify locally that all these services' base image has eBPF support? I'm not sure all these base images have the kernel that supports eBPF. Can you paste some screenshots?

Reply:
All of these are profiling envoy applications. 
![image](https://user-images.githubusercontent.com/3417650/167768367-f233089a-8811-4a74-852d-582e1b3d8abe.png)